data/example-data-postgres-book/src/main/java/io/americanexpress/data/book/DataBookApplication.java

```diff
@@ -0,0 +1,24 @@
+package io.americanexpress.data.book;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+
+/**
+ * DataBookApplication is the Spring Boot main class to start the data application.
+ */
+@SpringBootApplication
+@EnableConfigurationProperties
```

Comment:
I don't think we need `@EnableConfigurationProperties`. I believe `@SpringBootApplication` already has it by default.